### PR TITLE
Update PHM/CRIS Welsh translations

### PR DESCRIPTION
**Pull request by Yuyuutsu**
### What is the context of this PR?
Updates the translations for CRIS schema.
Trello card is [here](https://trello.com/c/5Ct4e8SF/5728-generate-update-welsh-schema-for-phm-cris-1-sp)
### How to review
Run make `translate-schemas` and ensure the change is identical.
You may want to download the `cris_0001.po` file from the Trello card yourself and translate the schema to test it is the same.

### CODE REVIEW

**Comment by Yuyuutsu** [comment]
[Generate/Update Welsh Schema for PHM/CRIS (1 SP)](https://trello.com/c/5Ct4e8SF/5728-generate-update-welsh-schema-for-phm-cris-1-sp)

File: poetry.lock [hunk 1]
```diff
@@ -252,7 +252,7 @@ graph = ["objgraph (>=1.7.2)"]
 
 [[package]]
 name = "eq-translations"
-version = "4.9.0"
+version = "4.8.0"
```
Comment (MebinAbraham) [inline poetry.lock:255]: The packages are already up-to-date, just your local one wasn't. You shouldn't revert the lock file and run `poetry update` locally. It has actually reverted it.

Comment (Yuyuutsu) [reply poetry.lock]: Oh hahaha. Okay its back to normal now 😌

**Review by katie-gardner — APPROVED** [review]
po file identical to trello one, json generated is the same, tested the survey and can see updated content coming through 👍

**Pull request merged**